queries/java/locals.scm

```diff
@@ -1,10 +1,37 @@
-; CREDITS @maxbrunsfeld (maxbrunsfeld@gmail.com)
-
+; SCOPES
 (class_declaration
-  name: (identifier) @name) @class
+   body: (_)) @scope
+(method_declaration) @scope
+(block) @scope
+; 3 captures for `if` - if+else, if only, else only
+(if_statement) @scope
+(if_statement
+  body: (_)) @scope
+(if_statement
+  alternative: (_)) @scope
+; 3 captures for`try` - try+catch, try only, catch only
+(try_statement) @scope
+(try_statement
+  body: (_) @scope)
+(catch_clause) @scope
+
+(for_statement) @scope
+(do_statement) @scope
```

Comment:
These dont have a `body` field ?

Reply:
nope i want to capture the whole thing. i tagged you in the relevant message on gitter.